python/tvm/autotvm/tophub.py

```diff
@@ -208,6 +208,10 @@ def load_reference_log(backend, model, workload_name, template_key):
     version = PACKAGE_VERSION[backend]
     package_name = "%s_%s.log" % (backend, version)
     filename = os.path.join(AUTOTVM_TOPHUB_ROOT_PATH, package_name)
+    # if fallback config file does not exist, download it from tophub
+    if not os.path.exists(filename):
```

Comment:
It would be better to move this logic after key checking.

Reply:
Thanks! Comments have been addressed